src/test/java/org/broadinstitute/hellbender/tools/exome/SegmentUtilsUnitTest.java

```diff
@@ -39,28 +43,16 @@ public void testReadAndWriteCalledIntervals() {
     }
 
     @Test
-    public void testMeanTargetCoverage() {
```

Comment:
Because the only uses of `meanTargetCoverage()` were in `testMeanTargetCoverage()`.